js/google_ima.js

```diff
@@ -1270,6 +1270,7 @@ require("../html5-common/js/utils/utils.js");
       var _IMA_SDK_onAdClicked = privateMember(function(adEvent)
       {
         _amc.adsClicked();
+        _amc.adsClickthrough();
```

Comment:
adsClicked and adsClickthrough sound the same.  Can you change adsClickthrough to adsClickthroughOpened?